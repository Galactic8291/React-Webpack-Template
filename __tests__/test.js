// Test Test
import React from 'react'
import { shallow } from 'enzyme'

import App from '../src/components/App'

describe('Hello', () => {
  it('HELLO WORLD', () => {
    expect(true).toBe(true)
  })

  it('HELLO Component', () => {
    const output = shallow(<App />)

    expect(output).toMatchSnapshot()
  })
})

