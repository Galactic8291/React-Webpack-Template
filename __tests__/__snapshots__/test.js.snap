// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello HELLO Component 1`] = `
<div>
  Hello World!!!
</div>
`;
